cs130_front/src/components/Listings/ListingList.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Text from '../Text/Text';
+import * as Fonts from '../../constants/Fonts';
+import * as Colors from '../../constants/Colors';
+import ListingItem from './ListingItem';
+
+const ListingList = (props) => {
+
+  return (
+    <>
+    <div style = {{fontFamily: Fonts.Primary}}>
```

Comment:
Maybe some flex centering or sm. Right now it seems left aligned since its shorter than the tabs above it. Could make width the full width of the main panel or the same length of the tabs.